scripts/prog15.rb

```diff
@@ -0,0 +1,18 @@
+def prime(n)
+    (2..n).step(1).each do |value|
+        c=0
+        (2..value/2).each do |e|
```

Comment:
This can be optimized further.. please see if you can reduce the loop further from `value/2`